### DISCUSSION TITLE
Change abstract formatting so that first line is not indented

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#4094] [Q&A]
I am wondering why the default formatting for abstracts in Quarto includes an indented first line. In my field (political science) the norm is block paragraph style but I think it is also the standard for top journals like [Science](https://www.science.org/doi/10.1126/sciimmunol.ade7953) and [Nature](https://www.nature.com/articles/s41586-022-05639-4).  While I know that it is possible to modify the abstract format by inserting some LaTeX code, e.g. `\noindent`, it might be more convenient if the default was no indent.

**Comment by mcanouil** [comment]
I would suggest to make an extension with your custom specification/needs.
To note, you can take inspiration from https://quarto.org/docs/journals/.

PS: changing this to "Q&A".

**Reply by eteitelbaum** [reply]
Ultimately that is what I did, but I wonder if there is a good reason for having the default be indented. Are there fields/journals where the abstract is indented in the first line?

**Reply by mcanouil** [reply]
The default is not about journals, it's about document.
The standard "rule" is to indent the first line of each paragraph, abstract is a paragraph, thus is being indented.
Journals have their own guidelines/templates about how references, abstract, authors, etc. need to be formatted which basically differs from the standard writing rules.
When using Quarto (or any markups) to target journals, it's better to download the (LaTeX) template when provided and use that directly or within an extension.

**Reply by eteitelbaum** [reply]
That's what I've been told, but my experience is that there are not that many templates available yet and they are not easy to make. So from the standpoint of being user-friendly and uptake/adoption it might be helpful to have defaults that people can use right out of the box. Rather than asking users to fish around for complicated solutions. Just my two cents.

**Reply by mcanouil** [reply]
Maybe but that's your point of view for your use case.
What about users that need/want first line indent in paragraphs?
Here no matter the choice, it won't satisfy everyone.

There are as many templates as there are journals (if Word/LaTeX templates are not provided by journals, there is not much to be done), how do you set the default to fit them all? "User-friendly" does not imply the default should satisfy everyone, thus the extension mechanism which comes with documentation, examples, and Quarto CLI command to get a starter format, _i.e._, that is "user friendly".
As for any tools (this includes Microsoft Word), you need to learn beyond the default to get exactly what you want.

**Reply by cscheid** [reply]
> That's what I've been told, but my experience is that there are not that many templates available yet and they are not easy to make.

I do I understand where @eteitelbaum is coming from. We're working on making both more templates and making extension easier! Unfortunately, this is a problem that only time can solve.

**Reply by eteitelbaum** [reply]
@cscheid Did you see all of the new templates made by Christopher Kenny?

https://github.com/christopherkenny

This is great! I did not think we would have so many political science and social science options available so soon.

I see that the APSR template is featured in the [offiical list](https://quarto.org/docs/extensions/listing-journals.html) of journal article templates. I hope more of them will be listed there soon.

**Comment by eteitelbaum** [comment]
> Maybe but that's your point of view for your use case. What about users that need/want first line indent in paragraphs? Here no matter the choice, it won't satisfy everyone.

That's why I was raising it as a discussion question. I have not seen a journal or working paper with an indented first line in the abstract. But maybe that is unique to my field. Are there users who want the indented first line? 
 
> There are as many templates as there are journals (if Word/LaTeX templates are not provided by journals, there is not much to be done), 

Actually I tried this with the LaTeX template for the top journal in my field (APSR) and it didn't import properly. I would like to convert this into an opportunity to create a Quarto template for that journal but it s proving to be a steep learning curve and I doubt many other users are going to want to go to those lengths. 

Have you had better luck with other journal templates? 

>how do you set the default to fit them all? "User-friendly" does not imply the default should satisfy everyone, thus the extension mechanism which comes with documentation, examples, and Quarto CLI command to get a starter format, _i.e._, that is "user friendly". As for any tools (this includes Microsoft Word), you need to learn beyond the default to get exactly what you want.

I think the idea behind Quarto is that it saves you time on the formatting because everything is pretty much automatic. When users now have to go into not just learning Quarto but then also go back to learn LaTeX and Quarto extensions, which are entirely separate workflows, it does not fulfill its promise of saving time. If the extensions were available and it was just a simple matter of plug and play then yes, I agree with you. But if people have to make their own templates it is going to be a harder sell. 

Like Carlos mentions, some of this will be resolved over time. And already tremendous progress has been made. My point is just that for basic things like indenting and line spacing,  which a lot of people want, there should be some easy fixes that you can implement in the YAML header  (and I think the team is working towards this).
